### DISCUSSION TITLE
Move requirements.txt to docs/requirements.txt

### DISCUSSION THREAD

**Discussion by CarliJoy** [pyscaffold/pyscaffold#339] [General]
If the requirements are only there to be used by read the docs, then move the file and change the settings in read the docs.
Because normally people (and pycharm) will assume that requirements.txt in the main folder are development or deploy requirements.

**Comment by CarliJoy** [comment]
It looks like this using PyCharm:
![image](https://user-images.githubusercontent.com/7687556/102539992-b6c84f00-40ae-11eb-9822-806e95a6fd4b.png)

**Comment by FlorianWilhelm** [comment]
Good point @CarliJoy. This seems to be a reminiscence of times when ReadTheDocs could not handle conda environments. A first WIP PR can be found under #340.

**Comment by julie777** [comment]
Why does readthedocs need a requirements.txt?

**Reply by abravalheri** [reply]
Docs have dependencies, for example they can use a different template package, a special sphinx plugin, etc...

`requirements.txt` is just a handy way of telling RTD to install these packages before trying to build the docs.

It does not have to be a `requirements.txt`. [RTD can use other methods other than pip](https://docs.readthedocs.io/en/stable/config-file/v2.html), e.g. conda, system packages via apt, etc..
